docs/walkthrough/editing-a-package.md

```diff
@@ -47,6 +49,18 @@ Use the `set` and `set_dir` commands to add individual files and whole directori
 
 
 ```python
+# Create test directories
+import quilt3
+from pathlib import Path
+from os import chdir
+TEST_DIR = "test_workflow"
+SUB_DIR = "subdir"
+
+# create test directories
+Path(TEST_DIR).mkdir(exist_ok=True)
+Path(TEST_DIR, SUB_DIR).mkdir(exist_ok=True)
+chdir(TEST_DIR) # %cd TEST_DIR/ if in Jupyter
```

Comment:
I think instead of doing this you should try using pytest auto fixture like this:
https://github.com/quiltdata/quilt/blob/053e5ca95549460756d88c2836d5389e9e9d962b/api/python/tests/conftest.py#L74-L84

That way tests are (somewhat) isolated from each other and don't depend on each other and the order of tests execution, files created during tests are created in temporary directories.

Currently if you run `poetry run pytest  --codeblocks  ../docs/walkthrough/installing-a-package.md` you get `scripts/` in root of `testdocs` which is not listed in `.gitignore`.

Reply:
Huh.  Okay, I will try adding a commented fixture to a doc file and see if it works.

Reply:
Ah, I realize the challenge. This documentation is extracted from a Python Notebook, where we explicitly *want* the user to create these subdirectories when running the notebook.  It is important the code snippets work there.  The "--codeblocks" argument, like the notebook, expects cells to be run in order, so we can count on it.

I have added 'scripts' to the `.gitignore` -- this seems to come from the package install in the tutorial, so there's no way for us to not create it.